spec/redux/actionCreators-test.js

```diff
@@ -165,118 +132,153 @@ describe("The actionCreators'", () => {
                done();
              }, 0);
            });
 
-        describe("and the execute function,", () => {
+        describe("and the execute function", () => {
 
           beforeEach((done) => {
-            compileResolve("the bytecode");
+            compileResolve('the bytecode');
             window.setTimeout(done, 0);
           });
 
           it(`dispatches a FINISH_EXECUTE and STORE_EDITOR_RESULT
               action once the bytecode is executed`,
              (done) => {
-               executeResolve("some result");
+               executeResolve('some result');
                window.setTimeout(() => {
                  expect(store.getActions()[6])
-                   .toEqual({type: "FINISH_EXECUTE", payload: "some result"});
+                   .toEqual({type: 'FINISH_EXECUTE', payload: 'some result'});
                  expect(store.getActions()[7])
-                   .toEqual({type: "STORE_EDITOR_RESULT", payload: "some result"});
+                   .toEqual({type: 'STORE_EDITOR_RESULT', payload: 'some result'});
 
                  done();
                }, 0);
              });
 
           it("dispatches a FAIL_EXECUTE action once the third promise is rejected",
              (done) => {
-               executeReject("some error");
+               executeReject('some error');
                window.setTimeout(() => {
                  expect(store.getActions()[6])
-                   .toEqual({type: "FAIL_EXECUTE", payload: "some error"});
+                   .toEqual({type: 'FAIL_EXECUTE', payload: 'some error'});
                  done();
                }, 0);
              });
         });
       });
     });
   });
+
+  describe("expandMoreMenu function", () => {
+    it("dispatches a EXPAND_MORE_MENU action", () => {
+      store.dispatch(actCreators.expandMoreMenu());
+      expect(store.getActions()[0]).toEqual({type: actType.EXPAND_MORE_MENU});
+    });
+  });
+
+  describe("collapseMoreMenu function", () => {
+    it("dispatches a COLLAPSE_MORE_MENU action", () => {
+      store.dispatch(actCreators.collapseMoreMenu());
+      expect(store.getActions()[0]).toEqual({type: actType.COLLAPSE_MORE_MENU});
+    });
+  });
+
+  describe("incrementFontSize function", () => {
+    it("dispatches a INCREMENT_FONT_SIZE action", () => {
+      store.dispatch(actCreators.incrementFontSize());
+      expect(store.getActions()[0]).toEqual({type: actType.INCREMENT_FONT_SIZE});
+    });
+  });
+
+  describe("expandMoreMenu function", () => {
+    it("dispatches a DECREMENT FONT SIZE action", () => {
+      store.dispatch(actCreators.decrementFontSize());
+      expect(store.getActions()[0]).toEqual({type: actType.DECREMENT_FONT_SIZE});
+    });
+  });
+
+  describe("stop function", () => {
+    it("dispatches a STOP_RUN action", () => {
+      store.dispatch(actCreators.stop());
+      expect(store.getActions()[0]).toEqual({type: actType.STOP_RUN});
+    });
+  });
```

Comment:
so is this the same unaltered code but moved to a different location in the file for organization reasons?